### ISSUE TITLE
"Something went wrong" when adding some preexisting series to Sonarr.

### ISSUE THREAD

**Issue by EddiYo** [tidusjar/PlexRequests.Net#105]
Tried with a few different series that have not started yet, like "Marseille", "Outcast" and "Hunters".

Got the same error when adding "Doctor Who (2005)" which I already have.All it feeds back to the requesting user is a "Something went wrong" message.

Here is the logfile of the latest attempt. 

![fatal](https://cloud.githubusercontent.com/assets/17574014/14062969/6185855e-f3b4-11e5-8c52-3c3913d66858.PNG)

**Comment by tidusjar** [comment]
This is a known 'issue' area. The application doesn't give great feedback at the moment but we are working on that. The issue in the screenshot above is because you are attempting to add to Sonarr, but it's rejecting it because it's all ready in sonarr.

**Comment by EddiYo** [comment]
Yeah, kinda figured as much. Did a quick search to see if there was any open issues on it, but did not find one right of the bat, so created one. 

It's more of an issue for the requesting users, than it is for me with access to the log files. The reason I noticed this, was that I tried to populate the request list with every new item coming out "soon" so that the users themselves would not need to request it. 

But I imagine this might be pretty low on the priority list :)
